addons/mail/static/tests/helpers/mock_server.js

```diff
@@ -2176,4 +2180,31 @@ MockServer.include({
             starred_counter: this._getRecords('mail.message', [['starred_partner_ids', 'in', user.partner_id]]).length,
         };
     },
+    /**
+     * Simulate the `notify_cancel_by_type` on `mail.thread` .
+     * Note that this method is overridden by snailmail module but not simulated here.
+     */
+    _mockMailThreadNotifyCancelByType(model, notificationType) {
+        // Query related mail messages and notifications
+        const notifMails = this._getRecords('mail.notification', [
+            ['notification_type', '=', notificationType],
+            ['notification_status', 'in', ['bounce', 'exception']],
+        ]).map(notification => ({
+            notif: notification,
+            mailMsg: this._getRecords('mail.message', [['id', '=', notification.mail_message_id]])[0]
```

Comment:
Ok I see now, well I suggest to just name this `message` then. `mail` prefix is overkill in such internal methods.